### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by marko-bekhta**

### CODE REVIEW

**Review by yrodiere — APPROVED** [review]
Someone's having fun? :grin: 

Thanks a lot, but remember you need to update the dependencies in all relevant projects as well, to the exact same version. Otherwise it will still work, but build will take longer (because they'll update dependencies).

File: update-gemfile.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 COMPOSE=podman-compose
 if ! which "$COMPOSE" 2>&1 1>/dev/null
 then
-	COMPOSE=docker-compose
```
Comment (yrodiere) [inline update-gemfile.sh:6]: This... is actually potentially dangerous. But ok if you tested it (I can't, I use podman).

Comment (marko-bekhta) [reply update-gemfile.sh:6]: yeah ... with `docker-compose` there seems to be an error now: https://github.com/quarkusio/quarkusio.github.io/pull/2056#issue-2435011818

**Comment by yrodiere** [comment]
> Thanks a lot, but remember you need to update the dependencies in all relevant projects as well, to the exact same version. Otherwise it will still work, but build will take longer (because they'll update dependencies).

Thanks, I saw https://github.com/hibernate/hibernate.org/pull/216 and https://github.com/hibernate/in.relation.to/pull/200 , but I'll need another one for https://github.com/beanvalidation/beanvalidation.org/pulls :)

**Pull request merged**